### DISCUSSION TITLE
robotidy --list output bug?

### DISCUSSION THREAD

**Discussion by szantotomi** [MarketSquare/robotframework-tidy#536] [General]
Hello!
If I get it right, the **robotidy --list** command should print the enabled/disabled transformers based on the currently valid config (from my _pyproject.toml_ file, for example.
If this is right, then there is some bug around :)
Thankfully, i've found in that other discussion that a complete config was linked, so that was used as a baseline (basically it's the same that i had before).
There, I've disabled a transformer by updating to **"DiscardEmptySections:enabled=False"**
Still, the output of _robotidy --list_ shows this:
┏━━━━━━━━━━━━━━━
┃ Name                       ┃ Enabled ┃
┡━━━━━━━━━━━━━━
│ AddMissingEnd              │ Yes     │
│ NormalizeSeparators        │ Yes     │
**│ DiscardEmptySections       │ Yes**     │
...

However in fact, the transformer is really disabled as my "sandbox" code is not touched when robotidy is executed.

**Comment by bhirsz** [comment]
Hi, somehow I didn't see this discussion at all.. so I apologise for late reply.

You're right, it's doesn't look it works correctly.. it certainly accept configuration from the cli (so running ``robotidy --configure DiscardEmptySections:enabled=False --list`` will print Enabled - No). But it does not read configuration from the file. I will take a look. Do you want to create issue for it? Just copy paste from this question is enough.

And I will think about way of generating configuration file with the default settings (so it's easy to reuse it enable/disable transformers or see what options are available).

**Reply by szantotomi** [reply]
sure, will create an issue in a moment :) thanks for having a look!

**Comment by Peter-St05** [comment]
I matched the same problem as @szantotomi 
Is this already fixed? I am using the latest version of Robotidy.

Additionally using --config <path-to-config> is not working well. Only the pyproject.toml works...

**Reply by bhirsz** [reply]
What version you are using, most recent one? I remember it was fixed but I will check

**Reply by Peter-St05** [reply]
I am using 4.14.0 with robotframework 7.0.1
